Sources/Networking/Documentation.docc/Documentation.md

```diff
@@ -0,0 +1,297 @@
+# Networking
+
+A networking layer using native `UrlSession` and Swift concurrency.
+
```

Comment:
adding some notes I miss overall:
typealiases we have for responses (tuples)
how storing and reusing stored api calls works (that there is an provider to read it from assets when adding whole folder in original sequence of calls)